tests/doc.md

```diff
@@ -184,51 +184,44 @@ An example test case file would look like this:
+    retval = ["$abort_exit_value"]
+    stdout = ["Reading $filename"]
     stderr = [
-        """{abort_error}
-error in {filename}
+        """$abort_error
+error in $filename
 """
     ]
 ```
 
 The first 6 lines are necessary boilerplate to pull in the necessary routines to
 run the actual tests (these are implemented in the module `system_tests` with
-the class `system_tests.Case` extending `unittest.TestCase`). When adding new
-tests one should choose a new class name that briefly summarizes the test. Note
-that the file name (without the extension) with the directory structure is
-interpreted as the module by Python and pre-pended to the class name when
-reporting about the tests. E.g. the file `regression/crashes/test_bug_15.py`
-with the class `OutOfBoundsRead` gets reported as
-`regression.crashes.test_bug_15.OutOfBoundsRead** already including a brief
-summary of this test.
-
-**Caution:** Always import `system_tests` in the aforementioned syntax and don't
-use `from system_tests import Case`. This will not work, as the `system_tests`
-module stores the suite's config internally which will not be available if you
-perform a `from system_tests import Case` (this causes Python to create a copy
-of the class `system_tests.Case` for your module, without reading the
-configuration file).
+the meta-class `system_tests.CaseMeta` which performs the necessary preparations
+for the tests to run). When adding new tests one should choose a new class name
+that briefly summarizes the test. Note that the file name (without the
+extension) with the directory structure is interpreted as the module by Python
+and pre-pended to the class name when reporting about the tests. E.g. the file
+`regression/crashes/test_bug_15.py` with the class `OutOfBoundsRead` gets
+reported as `regression.crashes.test_bug_15.OutOfBoundsRead` already including
+a brief summary of this test.
 
 In the following lines the lists `commands`, `retval`, `stdout` and `stderr`
-should be defined. These are lists of strings and must all have the same amount
+should be defined. These are lists of strings and must all have the same number
 of elements.
 
-The test suite at first takes all these strings and substitutes all values in
-curly braces with variables either defined in this class alongside (like
+The test suite at first takes all these strings and substitutes all values
+following a `$` with variables either defined in this class alongside (like
 `filename` in the above example) or with the values defined in the test suite's
 configuration file. Please note that defining a variable with the same name as a
 variable in the suite's configuration file will result in an error (otherwise
 one of the variables would take precedence leading to unexpected results). The
-substitution of values in performed using Python's string `format()` method and
-more elaborate format strings can be used when necessary.
+substitution of values in performed using the template module from Python's
```

Comment:
typo? "in performed", it looks like it should be "is performed"

Reply:
That was indeed a typo, I fixed it.